src/ruby/ext/grpc/extconf.rb

```diff
@@ -37,24 +37,11 @@
     ENV['LD'] = ENV['CC']
 end
 
-if RUBY_PLATFORM =~ /darwin/
-  ENV['AR'] = 'libtool'
-  ENV['ARFLAGS'] = '-o'
- end
-
 ENV['EMBED_OPENSSL'] = 'true'
 ENV['EMBED_ZLIB'] = 'true'
 ENV['EMBED_CARES'] = 'true'
 
 ENV['ARCH_FLAGS'] = RbConfig::CONFIG['ARCH_FLAG']
-if RUBY_PLATFORM =~ /darwin/
```

Comment:
Unless I'm missing something, all of these macos-specific options are still needed for people building the gem from source on macos hosts.

Reply:
:facepalm: Let me fix that.